src/pk_vm.c

```diff
@@ -1217,7 +1217,17 @@ static PkResult runFiber(PKVM* vm, Fiber* fiber) {
     }
 
     OPCODE(BIT_NOT):
-      TODO;
+    {
+      // Don't pop yet, we need the reference for gc.
+      Var val = POP();
```

Comment:
This will pop the value from the stack. And the comment says
> // Don't pop yet, we need the reference for gc.

This value shouldn't be popped, doing so will make us lose its reference, use `PEEK(-1)` to keep it on the stack till we are done using it, the `DROP()` below will pop it once we're done.

Reply:
Oh yes, you're right. I wasn't yet done (was still experimenting on something) but thanks for catching that. I appreciate, honestly!